src/utilities/generate_image.cxx

```diff
@@ -1,6 +1,6 @@
 /*
   Copyright (C) 2003-2011, Hammersmith Imanet Ltd
-  Copyright (C) 2018-2020, University College London
+  Copyright (C) 2018-2022, University College London
```

Comment:
this copyright can just be UCL 2022, as there will be essentially nothing left in the file.